src/pipecat/services/google/llm.py

```diff
@@ -419,7 +422,14 @@ def from_standard_message(self, message):
         role = message["role"]
         content = message.get("content", [])
         if role == "system":
-            self.system_message = content
+            # System instructions are returned as plain text
+            if isinstance(content, str):
+                self.system_message = content
+            elif isinstance(content, list):
+                # If content is a list, we assume it's a list of text parts, per the standard
```

Comment:
Noticed this bug as I was working. Here I back-ported the fix to the non-universal-context code path.